### PR TITLE
Merge template

### PR DESCRIPTION
**Pull request by jenniferboedker**
**Purpose of Change**
This PR introduces the Qube template into core-utils-lib. 

**Changes**
Travis was removed and replaced with GitHub actions, the parent pom was removed from the POM

### CODE REVIEW

**Comment by jenniferboedker** [comment]
[port core-utils-lib into qube](https://trello.com/c/IICMchvC/51-port-core-utils-lib-into-qube)

File: .github/workflows/generate-reports.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install git
+        run: sudo apt-get install git
+
+      - name: Set up JDK 1.11
```
Comment (jenniferboedker) [inline .github/workflows/generate-reports.yml:24]: should be java 8 everywhere? How can I introduce java 8 with qube?

**Comment by jenniferboedker** [comment]
FYI: The test failures have also been there in the development branch

File: .github/workflows/sync_project.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          uses: actions/setup-python@v1
+          with:
+            python-version: 3.8
+
+        - name: Install qube
+          run: pip install qube
+
+        - uses: actions/checkout@v2
+          with:
+              fetch-depth: 0
```
Comment (wow-such-code) [inline .github/workflows/sync_project.yml:22]:
```suggestion
              fetch-depth: 0
              token: '${{ secrets.JOHNNY_Q5_REPORTS_TOKEN}}'
```

File: .github/workflows/build_package.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (wow-such-code) [inline .github/workflows/build_package.yml:14]:
```suggestion
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
```

File: .github/workflows/generate-reports.yml (hunk 1 above)
Comment (wow-such-code) [reply .github/workflows/generate-reports.yml:24]: I think it should be for now. You can just change 1.11 to 1.8

File: .github/workflows/generate-reports.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (wow-such-code) [inline .github/workflows/generate-reports.yml:27]:
```suggestion
          java-version: 1.8
```

File: .github/workflows/java_checkstyle.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (wow-such-code) [inline .github/workflows/java_checkstyle.yml:14]:
```suggestion
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
```

File: .github/workflows/nexus-publish-release.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
```
Comment (wow-such-code) [inline .github/workflows/nexus-publish-release.yml:22]:
```suggestion
    - name: Set up JDK 1.8
      uses: actions/setup-java@v1
      with:
        java-version: 1.8
```

File: .github/workflows/nexus-publish.yml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (wow-such-code) [inline .github/workflows/nexus-publish.yml:23]:
```suggestion
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
```

File: .github/workflows/run_tests.yml [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (wow-such-code) [inline .github/workflows/run_tests.yml:16]:
```suggestion
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
```

**Review by wow-such-code — CHANGES_REQUESTED** [review]
lgtm, but some small changes needed

File: .github/workflows/sync_project.yml (hunk 2 above)
Comment (wow-such-code) [reply .github/workflows/sync_project.yml:22]: Disclaimer: I'm assuming here that's needed for all projects, Sven added that to the projectwizard.

File: .github/workflows/generate-reports.yml (hunk 1 above)
Comment (jenniferboedker) [reply .github/workflows/generate-reports.yml:24]: Perfect, thanks!

File: .github/workflows/sync_project.yml (hunk 2 above)
Comment (jenniferboedker) [reply .github/workflows/sync_project.yml:22]: Nice catch! I am not sure maybe it needs to be specified for this repo also?

Comment (wow-such-code) [reply .github/workflows/sync_project.yml:22]: I think so, since it seems to have to do with checkout and is not portlet-specific.

**Comment by jenniferboedker** [comment]
We still need to remove Travis from this repo, should I open an issue for this or should this be part of this PR as it is required because of the porting?

**Review by wow-such-code — APPROVED** [review]
well-spotted

**Pull request merged**